src/python/gudhi/sktda/kernel_methods.py

```diff
@@ -0,0 +1,202 @@
+"""
+@author: Mathieu Carriere
```

Comment:
This appears verbatim in the sphinx documentation, probably not what you want. Current source files in gudhi seem to handle it differently. On the other hand, I don't see the author for the RipsComplex reference page. We could set `show_authors=True` in doc/conf.py, and add `.. codeauthor:: Me me me` or `.. sectionauthor:: Myself` in the .rst file. Or don't use automation, specify the author manually in the ref page the same way as the user page, and specify the author in the source file in a way that doesn't show in sphinx, as we do in most files currently.